01-wasmcloud_basics-pathway.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "wasmcloud basics",
```

Comment:
Same as the other review...throughout use wasmCloud as a standalone noun, and only use `wasmcloud` when referring to the binary.

Reply:
Will to top level stuff in a separate PR so I dont have to rebase all the branches.